.github/workflows/check-templates.yml

```diff
@@ -365,13 +365,18 @@ jobs:
         id: branch
         run: |
           repo_name=$(basename "${{ github.repository }}")
-          echo "source-branch=${{ env.source-branch }}_${repo_name}@devel" >> $GITHUB_OUTPUT
+          source_br="${{ env.source-branch }}_${repo_name}@devel"
```

Comment:
@cicdguy : since this PR includes several changes I just add some comments ! here I added this change for pharmaverseadam datasets creation (since we push updates to a new branch, I just make a check before to see if the branch exists - if not we do not trigger the "commit and push changes" step (then we avoid error cases that we often have since pharmaverseadam maintainers might sometimes forget to merge some pending PRs..)
I also thought creating unique branch but would be super messy I guess

Reply:
Oh yes indeed you're right :) just from some old runs I guess I just forgot to delete it 🙈

Reply:
(done)